targets/TARGET_TOSHIBA/mbed_rtx.h

```diff
@@ -25,10 +25,6 @@
 #ifndef INITIAL_SP
 #define INITIAL_SP                        (0x20080000UL)
 #endif
-#ifdef MBED_CONF_RTOS_MAIN_THREAD_STACK_SIZE
-#undef MBED_CONF_RTOS_MAIN_THREAD_STACK_SIZE
-#endif
-#define MBED_CONF_RTOS_MAIN_THREAD_STACK_SIZE 3072
```

Comment:
Note: This affects only TMPM46B - see a few lines above.
We remove this macro-redefinition and override it properly in rtos/source/TARGET_CORTEX/mbed_lib.json.